Pipfile

```diff
@@ -7,12 +7,12 @@ name = "pypi"
 dj-database-url = "*"
 django-cors-headers = "*"
 gunicorn = ">=19.5.0"
-Django = ">=1.11.15"
+Django = "==1.11.21"
```

Comment:
As `2` is greater than or equal to `1` this was upgrading Django. That's for a separate PR so pinned to this version for now.